doc/constants_pre-re.md

```diff
@@ -5807,7 +5807,7 @@
 - `ModDeath`: [param]
 
 
-## Skills (db/pre-re/skill_db.txt)
+## Skills (db/pre-re/skill_db.conf)
```

Comment:
These are auto generated files you should not change them manually but rather update the code in `plugins/constdb2doc.c`